api/certificate.go

```diff
@@ -13,6 +13,22 @@ import (
 	"github.com/tsuru/rpaas-operator/internal/pkg/rpaas"
 )
 
+func deleteCertificate(c echo.Context) error {
+	manager, err := getManager(c)
+	if err != nil {
+		return err
+	}
+
+	instance := c.Param("instance")
+	certName := c.Param("name")
```

Comment:
I think you cannot use this parameter since you didn't define it on routing expression.